.github/workflows/merlin.yml

```diff
@@ -369,26 +367,45 @@ jobs:
       INGRESS_HOST: "127.0.0.1.nip.io"
       MERLIN_CHART_VERSION: 0.11.4
       E2E_PYTHON_VERSION: "3.10.6"
+      K3S_VERSION: v1.26.7-k3s1
     steps:
       - uses: actions/checkout@v2
         with:
           path: merlin
-      - uses: actions/setup-go@v2
-        with:
-          go-version: ${{ env.GO_VERSION }}
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ env.E2E_PYTHON_VERSION }}
       - uses: actions/cache@v3
         with:
           path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('**/requirements.txt') }}
+          key: ${{ runner.os }}-pip-3.10-${{ hashFiles('**/requirements.txt') }}
```

Comment:
This change is also for debugging, yeah?

Reply:
No, it's not. e2e-test job does not use matrix.python-version but just python 3.10, so this modification fixed it.